frontend/plugins/sales_ui/src/modules/deals/cards/components/detail/overview/tags/SelectTags.tsx

```
@@ -0,0 +1,122 @@
+    variables: {
+      type: 'sales:deal',
+    },
+  });
+
+  const { giveTags, loading: giveTagsLoading } = useGiveTags();
+
+  const treeData = useMemo(() => buildTree(tags), [tags]);
+
+  const toggleExpand = (id: string) => {
+    const copy = new Set(expandedIds);
+    copy.has(id) ? copy.delete(id) : copy.add(id);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)